### DISCUSSION TITLE
德州撲克

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#617] [General]
我也喜歡打德州撲克，那我下次也想跟訓練過的AI PK一下
https://www.1111.com.tw/news/jobns/149382

**Comment by magicmicheal** [comment]
感覺跟機率有關的遊戲，人類應該都很難戰勝AI，連圍棋那種一步步都緊緊相連的機率，人腦都沒有可能戰勝了

**Comment by Xuewen628** [comment]
之前不是有AI和世界冠軍玩西洋棋並戰勝冠軍的新聞嗎
那時掀起了一波擔憂未來AI發展的話題呢

**Comment by awehao** [comment]
想當年，高中的時候在學校打德州撲克跟橋牌，人的思考思路，每個人的思路都有特定的模式，所以打久了都能稍微猜到對方的策略，不過若是AI普及，每次跟AI打，都能產生新的思路，不過這也喪失了像我們這種把打牌當休閒娛樂的人玩牌的宗旨，快樂享受才是最重要的，所以老話一句，我忘記怎麼講了，反正就是，AI不是不好，看你如何使用以及自己的需求

**Comment by awkward-willy** [comment]
AI 相比人類能更快速也更深遠的想到各式各樣接下來可能出現的牌面，因此隨著他們的不斷發展，我們終將難以在這些遊戲上贏過 AI。但在玩這些遊戲時，我覺得應該注重的是整體的過程，和友人開心地遊玩即可，不必總是想著輸贏。

**Comment by manipulateed** [comment]
AI分析牌面和機率的準確度和速度肯定快於人類，但AI還很公式化，一些心理戰的層面可能就很難對AI實施，大幅降低人類贏的難度。
